docs/source/open-software-summer-school/2026/large-data.md

```diff
@@ -0,0 +1,66 @@
+(track-large-array-data-2026)=
+# Track: Large Array Data
+
+In many areas of research, large array datasets have become important and wide-spread. Examples include whole-organ images (e.g. light-sheet, HiP-CT), Neuropixels data, and long behavioural videos — to name just a few. Processing, analyzing and visualising these datasets is made significantly more complex and slow because they do not fit into a typical computer's memory and take up a lot of disk space.
+
+The use of compression algorithms and "chunked" file formats (which favour parallel processing) enable us to deal with these challenges effectively, but doing this well requires careful consideration of the trade-offs between read and write speeds, disk space used, and portability.
+
+This track will cover the necessary context needed to understand the computational challenges posed by large array datasets, and how to address these challenges using open-source software tools. All sessions will include a significant hands-on component.
```

Comment:
Could we make it a bit clearer what participants will be able to by the end of the week? "Process a 1TB functional imaging dataset on their laptop" or things like that?